src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -39,10 +39,18 @@
                   text: >
                     az vm create -n my_vm_name -g myrg --admin-username myadmin --admin-password Password@1234
                      --public-ip-address "" --image Win2012R2Datacenter
-                - name: Create a simple Windows Server VM with public IP address and DNS entry
+                - name: Create a simple Ubuntu VM with public IP address, DNS entry and 2 data disk(10GB, 20GB)
                   text: >
                     az vm create -n my_vm_name -g myrg --admin-username myadmin --admin-password Password@1234
-                    --public-ip-address-dns-name my_globally_unique_vm_dns_name --image Win2012R2Datacenter
+                    --public-ip-address-dns-name my_globally_unique_vm_dns_name --image ubuntults --data-disk-sizes-gb 10 20 --size Standard_DS2
+                - name: Create a VM with unmanaged os disk by using image blob uri
+                  text: >
+                    az vm create -g myrg -n my_vm_name --image https://account123.blob.core.windows.net/Images/my_vhd-osDisk.vhd
+                    --os-type linux --admin-username myadmin --admin-password Password@1234
+                - name: Create a VM by attaching to an existing specialized managed os disk
+                  text: >
+                    az vm create -g myrg -n my_vm_name --managed-os-disk my-os-disk
```

Comment:
This is a good question. I have updated the examples. Let me know if you have any questions.  In short , the `--image` will be a image resource id or name (if it is under the same group). All others stay the same